### PR TITLE
Update to Angular 7

### PR DESCRIPTION
**Pull request by flavens**
Update BEOL to Angular 7 following the Angular Update Guide.

### CODE REVIEW

**Comment by flavens** [comment]
[Update BEOL app to angular 7](https://trello.com/c/FEHYpiAV/62-update-beol-app-to-angular-7)

**Comment by tobiasschweizer** [comment]
@flavens I can confirm the application works with Angular 7. I suggest that you publish the Knora-ui 7 modules and we then integrate them on this PR and check again. 

@SepidehAlassi Once the Knora-ui 7 modules are ready and integrated on this branch (`package.json`), could you do a check as well? @flavens will let you know once the Knora-ui 7 modules are ready and help with `package.json`.

**Comment by flavens** [comment]
@tobiasschweizer I've had the kui module v7.0.0 in package.json + update travis.yml - I kept a copy of the file with the yalc command lines

**Comment by SepidehAlassi** [comment]
@tobiasschweizer I checked this. all seems to be fine.

File: .dockerignore [hunk 1]
```diff
@@ -2,5 +2,4 @@
 /node_modules
 /yarn.lock
 /dist
-/.yalc
```
Comment (tobiasschweizer) [inline .dockerignore:5]: @flavens Does this have to be undone?

**Review by tobiasschweizer — APPROVED** [review]
Great!

**Pull request merged**